blazor/introduction.md

```diff
@@ -79,8 +79,10 @@ padding-top: 20px;
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/pivot-table/getting-started">Pivot Table</a></div>
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/treegrid/getting-started">TreeGrid</a></div>
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/multicolumn-combobox/getting-started">Multicolumn ComboBox</a></div>
+        <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/data/getting-started-with-web-app">Data Manager</a></div>
```

Comment:
Data Manager is not separate component

Reply:
Hi @rajendranr-5483 ,
I removed this component from common UG documentation.
Regards,
@Backiaraj